### PR TITLE
Updating protractor to 4.0.10

### PR DESCRIPTION
**Pull request by jourdanrodrigues**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "event-stream": "3.1.5",
     "gulp-util": "3.0.1",
     "lodash": "2.4.1",
-    "protractor": "4.0.9"
+    "protractor": "4.0.10"
```
Comment (cnishina) [inline package.json]: Would it be better to have `"protractor": "~4.0.10"` to adopt all hot fixes.

**Comment by jourdanrodrigues** [comment]
Well observed. Should've seen it. Just perfomed an update on the commit. :)

**Comment by rochejul** [comment]
I prefer not :  in a production context, this is not recommended to use wildcard version to avoid fetching new issues and to guarantee each developer has got the same version

Moreover the project uses shrinkwrap (you have not update it) which made a snaspshot of tje node modules folder for the previous reasons.

Plus, the goal of the pull request is to update protractor to 4.0.10

Why is the need  ?  What kind of enhancements brought this version  ? 

Many thanks

**Comment by jourdanrodrigues** [comment]
Sorry for the delay.

I'm doing some tests to figure out how to solve [issue #27](https://github.com/rochejul/gulp-angular-protractor/issues/27), so, this pull request isn't complete yet.

Updating Protractor seemed to partially resolve the issue: it runs eventually, but is too much intermittently to establish a pattern.

Continuing studying the case here. :smile:

**Comment by cnishina** [comment]
In addition to updating protractor, because this is shrinkwrapped, you might want to also update webdriver-manager to 10.2.6. It has fixes for downloading the latest chromedriver.

**Comment by rochejul** [comment]
@cnishina @jourdanrodrigues I will plan an update to protractor and webdriver-manager

Many thanks for your feedback

**Pull request closed without merging**